ts/test/ load-models.test.ts

```diff
@@ -0,0 +1,26 @@
+import { loadModels } from "../src/sequelize_schema";
+import {
+  AutoIncrement,
+  Column,
+  DataType,
+  Model,
+  PrimaryKey,
+  Table,
+} from "sequelize-typescript";
+
+@Table({ tableName: "email", schema: "public" })
+class Email extends Model {
+  @PrimaryKey
+  @AutoIncrement
+  @Column(DataType.INTEGER)
+  id!: number;
+}
+
+describe("loadModels", () => {
+  it("should load models correctly for schema identifier", () => {
+    const expectedOutput =
+      "CREATE TABLE IF NOT EXISTS `public.email` (`id` INTEGER auto_increment , `createdAt` DATETIME NOT NULL, `updatedAt` DATETIME NOT NULL, PRIMARY KEY (`id`)) ENGINE=InnoDB;\n";
```

Comment:
```suggestion
      "CREATE TABLE IF NOT EXISTS `public`.`email` (`id` INTEGER auto_increment , `createdAt` DATETIME NOT NULL, `updatedAt` DATETIME NOT NULL, PRIMARY KEY (`id`)) ENGINE=InnoDB;\n";
```

It should quote schema and table separately, not as one, right?

Reply:
you are right, but is the the code of `sequelize` there is not much I can do 😢